locale/af.po

```diff
@@ -13739,7 +13739,7 @@ msgid "&Generate"
 msgstr "&Genereer"
 
 #: src/menus/PluginMenus.cpp
-msgid "Add / Remove Plug-ins..."
+msgid "Plugin Manager"
```

Comment:
The second commit in this sequence is quite unnecessary!  Remove it.

As part of the release process, we will automatically update the .po files using the GNU gettext family of tools (msgfmt, etc.)

Reply:
Commit dropped.